src/openfermion/measurements/_prony_test.py

```diff
@@ -0,0 +1,51 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+""" Tests for _prony.py"""
+import unittest
+from ._prony import prony
+import numpy
+
+
+class PronyTest(unittest.TestCase):
```

Comment:
We're moving to using just pytest and not unittest. So you don't need to encapsulate the tests in a class.

Reply:
Fixed - I changed all the self.assertX statements to just assert statements, this is correct, right?